back-office/frae-bo-maintenance.html

```diff
@@ -0,0 +1,98 @@
+  <script src="assets/javascripts/govuk-admin-template/modules/auto_show_modal.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/auto_track_event.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/checkbox_toggle.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/confirm.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/filterable_table.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/fixed_table_header.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/radio_toggle.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/toggle.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template/modules/track_click.js?body=1"></script>
+  <script src="assets/javascripts/govuk-admin-template.js?body=1"></script>
+
+  <link rel="shortcut icon" type="image/x-icon" href="assets/images/govuk_admin_template/favicon-development.png" />
+
+  <link rel="stylesheet" media="all" href="assets/stylesheets/govuk_admin_template.css?body=1" />
+
+  <!-- CSS resets and fixes for Bootstrap 3 in IE7 -->
+  <!--[if lte IE 7]>
+      <link rel="stylesheet" media="screen" href="assets/stylesheets/bootstrap-ie7.css?body=1" />
+    <![endif]-->
+  <!--[if lte IE 8]>
+      <script src="assets/javascripts/govuk-admin-template/vendor/html5.js?body=1"></script>
+      <script src="assets/javascripts/govuk-admin-template/vendor/respond.min.js?body=1"></script>
+      <script src="assets/javascripts/lte-ie8.js?body=1"></script>
+    <![endif]-->
+</head>
+
+<body class="environment-development">
+
+  <header class="
+      navbar
+      navbar-default
+      navbar-inverse
+      navbar-static-top
+      environment-indicator
+      add-bottom-margin" role="banner">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-toggle" data-toggle="collapse" data-target="header .navbar-collapse">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </a>
+        <a class="navbar-brand" href="/">Flood Risk Activity Exemptions Admin System</a>
+        <div class="environment-label">
+          Development
+        </div>
+      </div>
+    </div>
+  </header>
+  <section class="container">
+    <main role="main">
+      <header>
+        <h1>Service unavailable</h1>
+      </header>
+      <div class="article-container">
+        <article role="article" class="group">
+          <div class="inner text">
+
+            <p>We're carrying out some work on the flood risk activity exemptions registration service.</p>
+
+            <p>Sorry about this. We’ll have it back up and running as soon as we can.</p>
```

Comment:
I have a feeling that `'` is `&#39;` and part of the code HTML character set, but `’` is font specific. So personally, I'd use the former